internal/strict/strict.go

```diff
@@ -147,23 +167,37 @@ var (
 )
 
 // ValidateControlNames validates that the given control names are valid.
-func (controls Controls) ValidateControlNames(strictControlNames []string) error {
+func (controls Controls) ValidateControlNames(strictControlNames []string) (string, error) {
+	completedControls := []string{}
 	invalidControls := []string{}
 	validControls := controls.Names()
 
 	for _, controlName := range strictControlNames {
-		if !util.ListContainsElement(validControls, controlName) {
+		control, ok := controls[controlName]
+		if !ok {
 			invalidControls = append(invalidControls, controlName)
+			continue
+		}
+
+		if control.Status == StatusCompleted {
+			completedControls = append(completedControls, controlName)
 		}
 	}
 
+	var warning string
+	if len(completedControls) > 0 {
+		warning = fmt.Sprintf("The following strict control(s) are already completed: %s. Please remove any completed strict controls, as setting them no longer does anything. For a list of all ongoing strict controls, and the outcomes of previous strict controls, see https://terragrunt.gruntwork.io/docs/reference/strict-mode",
+			strings.Join(completedControls, ", "))
+	}
+
+	var err error
 	if len(invalidControls) > 0 {
-		return fmt.Errorf("%w\nInvalid value(s):\n- %s\nAllowed value(s):\n- %s",
+		err = fmt.Errorf("%w\nInvalid control(s):\n- %s\nAllowed control(s):\n- %s",
```

Comment:
```suggestion
		err = errors.Errorf("%w\nInvalid control(s):\n- %s\nAllowed control(s):\n- %s",
```

We should use our `internal/errors` package everywhere, otherwise we will lose the stack trace.